flax/version.py

```diff
@@ -13,5 +13,5 @@
 # limitations under the License.
 
 """Current Flax version at head on Github."""
-__version__ = "0.6.9"
+__version__ = "0.6.7"
```

Comment:
Shouldn't this be `0.6.10`?

Reply:
Yes, no idea why I put this.
Fixed, thanks for catching this!